index.html

```diff
@@ -0,0 +1,45 @@
+<!DOCTYPE html>
+<html lang="en" dir="ltr">
+
+<head>
+  <meta charset="utf-8">
+  <title>Intention Timer</title>
+  <link rel="preconnect" href="https://fonts.gstatic.com">
+  <link href="https://fonts.googleapis.com/css2?family=Montserrat:wght@300;400&display=swap" rel="stylesheet">
+  <link rel="stylesheet" href="./styles.css">
+</head>
+
+<body>
+  <header>
+    <h1>INTENTION TIMER</h1>
+  </header>
+  <main>
+    <section class="left-container">
+      <h2>New Activity</h2>
+      <form>
+        <p class="select-category"> Select a category:</p>
+        <div class="button-container">
+          <button type="button" name="button"> <img class="button-icon" src="./assets/study.svg" alt="lightbulb"><br>Study</button>
+          <button type="button" name="button"><img class="button-icon" src="./assets/meditate.svg" alt="lotus flower"><br>Meditate</button>
+          <button type="button" name="button"><img class="button-icon" src="./assets/exercise.svg" alt="running shoe"><br>Exercise</button>
+        </div>
+        <p>What would you like to acomplish during this time?<br><input class="accomplish-input" type="text" name="" value=""></p>
+        <div class="minutes-seconds-container">
+          <p>Minutes<br><input type="text" name="" value=""></p>
+          <p>Seconds<br><input type="text" name="" value=""></p>
+        </div>
+        <button class="start-activity-button" type="button" name="button">START ACTIVITY</button>
+      </section>
+    </form>
```

Comment:
This closing tag should be above the section closing tag.

Reply:
Resolved.